frontend/src/hooks/useActions.js

```diff
@@ -41,17 +40,30 @@ const useActions = (registerHotkeys = false) => {
   const upsertProject = useProjectsStore(s => s.upsertProject)
   const moveView = useViewStore(s => s.moveViewPosition)
   const createState = useProjectStore(s => s.createState)
-  const createComment = useProjectStore(s => s.createComment)
-  const createTransition = useProjectStore(s => s.createTransition)
   const screenToViewSpace = useViewStore(s => s.screenToViewSpace)
   const setTool = useToolStore(s => s.setTool)
   const project = useProjectStore(s => s.project)
   const updateProject = useProjectStore(s => s.update)
-
+  const insertGroup = useProjectStore(s => s.insertGroup)
+  const addTemplate = useTemplatesStore(s => s.upsertTemplate)
   const navigate = useNavigate()
 
   // TODO: memoize
   const actions = {
+    CREATE_TEMPLATE: {
+      hotkey: { key: 'j', meta: true },
+      handler: () => {
+        if (selectedStatesIds.length === 0 && selectedCommentsIds.length === 0 && selectedTransitionsIds.length === 0) {
```

Comment:
Might be better to move this into the disabled field

Reply:
Is there a way I can use this command for testing if it's disable? If not, I'm going to leave it as not disabled for now so that I can keep testing it. By the time this is merged to dev the UI will hopefully be merged so it shouldn't need to be disabled (although if it wasn't finished when merging I would make sure it's disabled)

Reply:
The logic will mostly stay the same, just means the UI button will be greyed out if nothing is selected (So you should be able to keep testing it)
```ts
disabled: () => selectedStatesIds.length === 0 && selectedCommentsIds.length === 0 && selectedTransitionsIds.length === 0,
handler: () => {
  // Normal code here, no need to check if stuff is selected anymore
```

Reply:
Oh right, I thought you meant disable it entirely but that makes more sense